CAIPs/caip-25.md

```diff
@@ -16,110 +16,90 @@ CAIP-25 defines an authorization procedure for a chain agnostic provider to inte
-It defines a lightweight protocol for negotiating and persisting authorizations during a session managed by a provider construct.
+This proposal defines a standard procedure for decentralized applications to interface with chain agnostic cryptocurrency wallets and other user agents that govern identities (including accounts) across multiple cryptographic systems. It specifies a lightweight protocol for negotiating and persisting authorizations during a session managed by a provider construct.
 
 ## Motivation
 
-The motivation comes from the lack of standardization across blockchains to expose accounts and define the expected JSON-RPC methods to be used by an application through a provider connecting to a signer or other user agent.
+The lack of standardization across blockchains for exposing accounts and defining expected JSON-RPC methods has created inconsistencies. CAIP-25 addresses this by enabling a unified, session-based interface between applications and wallets.
 
 ## Specification
 
 ### Language
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" written in
-uppercase in this document are to be interpreted as described in [RFC
-2119][]
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are interpreted as described in [RFC-2119][].
 
 ### Definition
 
 #### Session Lifecycle
 
-The session is proposed by a caller and the response by the respondent is used as the baseline for an ongoing session between the two parties.
-- When a wallet responds with a success response containing a `sessionId` (an entropic [identifier][CAIP-171]), the properties and authorization scopes that make up the session should be persisted and tracked over the life of the session by both parties in a discrete data store.
-- When the wallet does not provide a `sessionId` in its initial response, the wallet MUST persist and track the properties and authorization scopes that make up the session.
-The caller is not expected to persist session data or even a `sessionId`.
-Note that wallets NOT returning `sessionId`s MUST implement additional methods and notifications to handle the full lifecycle of the session:
-    * [`wallet_getSession`][CAIP-312] to enable the caller to query for the current status of the session at any time.
-    * [`wallet_revokeSession`][CAIP-285] to explicitly end the session
-    * [`wallet_sessionChanged`][CAIP-311] to notify caller of updated session authorizations.
+The session is proposed by a caller and accepted by the respondent. The respondent may return a `sessionId` which both parties then persist along with session properties and authorization scopes.
```

Comment:
should we atleast reference [CAIP-316](https://chainagnostic.org/CAIPs/caip-316)
```suggestion
The session is proposed by a caller and accepted by the respondent. The respondent may return a `sessionId` which both parties then persist along with session properties and authorization scopes. See [CAIP-316][] for guidance on session lifecycles with and without `sessionId`s.
```